.circleci/config.yml

```diff
@@ -389,74 +389,80 @@ jobs:
   # Publish docker packages
   "publish/docker-dev":
     docker:
-      - image: docker:stable
+      - image: cimg/base:stable
```

Comment:
Switched to cimg/base:stable as it already provides `docker buildx` command for us, when a 20.10.x version of docker is requested.